src/main/java/calculator/Numbers.java

```diff
@@ -0,0 +1,17 @@
+package calculator;
+
+import java.util.List;
+
+public class Numbers {
+
+    private final List<Number> numbers;
+
+    public Numbers(List<Number> numbers) {
+        this.numbers = numbers;
+    }
+
+    public Number sum() {
+        return numbers.stream().reduce(Number.ZERO_NUMBER, Number::add);
+    }
```

Comment:
```suggestion
    public Number sum() {
        return numbers.stream()
                .reduce(Number.ZERO_NUMBER, Number::add);
    }
```
연산마다 개행을 주면 가독성을 높일 수 있습니다 :)